test-infra/prowjobs/unittests/main.sh

```diff
@@ -44,8 +44,26 @@ for TARGET in "${TARGETS[@]}"; do
   fi
 
   # Output test report.
+  cat test.out
   cat test.out | go-junit-report > ${ARTIFACTS}/${TARGET//\//_}_report.xml
   rm test.out
 done
 
+# Execute all unittests.sh scripts in their directories.
+# Each script should APPEND coverage data to GOCOVPATH or PYCOVPATH.
+RET=0
+find . -type f -name "unittests.sh" | while read script; do
```

Comment:
`while read` loops consume the stdin file descriptor and can wreak havoc on scripts that expect input. just an FYI, a `for` loop or `xargs` are typically preferred.

Reply:
good to know! This file will get reverted, but I'll keep that info for future work :)